multiversx_sdk_cli/cli_config.py

```diff
@@ -93,3 +98,21 @@ def list_configs(args: Any):
         if config_name == data.get("active", "default"):
             config_name += "*"
         print(config_name)
+
+
+def delete_config(args: Any):
+    config_file = config.resolve_config_path()
+    if not config_file.is_file():
+        logger.info(f"Config file not found. Aborting...")
+        return
+
+    confirm_continuation(config_file)
+    os.remove(config_file)
+    logger.info("Successfully deleted the config file")
+
+
+def confirm_continuation(file: Path):
```

Comment:
The function can be renamed to contain the specific action (e.g. deletion of the file), or it can be changed to receive the message as a parameter - and then it will be a more generic function (e.g. maybe for `ux.py`).

Reply:
moved the function in `ux.py` and made it more generic.